packages/react-meteor-data/withTracker.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { memo } from 'react';
+import useTracker from './useTracker.js';
+
+export default function withTracker(options) {
+  return Component => {
+    const expandedOptions = typeof options === 'function' ? { getMeteorData: options } : options;
+    const { getMeteorData, pure = true } = expandedOptions;
+
+    function WithTracker(props) {
+      const data = useTracker(() => getMeteorData(props) || {}, [props]);
+
+      if (Package.mongo && Package.mongo.Mongo && data) {
+        Object.keys(data).forEach((key) => {
+          if (data[key] instanceof Package.mongo.Mongo.Cursor) {
+            console.warn(
+              'Warning: you are returning a Mongo cursor from withTracker. '
+              + 'This value will not be reactive. You probably want to call '
+              + '`.fetch()` on the cursor before returning it.'
+            );
+          }
+        });
+      }
+
+      return <Component {...{ ...props, ...data }} />;
```

Comment:
Just out of curiosity, is there a special reason against `{...props} {...data}`?

Reply:
not really :-)

Reply:
changed to `{...props} {...data}`